osu.Game/Overlays/BeatmapSet/Scores/ScoresContainer.cs

```diff
@@ -97,6 +104,20 @@ public IReadOnlyList<ScoreInfo> Scores
             }
         }
 
+        private APILegacyUserTopScoreInfo userScore;
+
+        public APILegacyUserTopScoreInfo UserScore
```

Comment:
This feels *really* awkward. Why is this here, and why does it cancel the request?

Reply:
Fixed, should be better now